### ISSUE TITLE
VuSection

### ISSUE THREAD

**Issue by nyckysoft** [Negrero/tachograph-structure#4]
Hello.

in VuSizes the dimension of VUCALIBRATIONRECORD should be 167 instead of 164. There is an error in the specifications. If you add all the components you will have 167 and not 164 as they noted. All the best.

**Comment by AskmeKolaric** [comment]
@nyckysoft Where did you find this information?

**Comment by nyckysoft** [comment]
According to the specs for the tachographs a calibration reccord contains the following informations

CALIBRATIONPURPOSE(1),
WORKSHOPNAME_VUCALIBRATIONDATA(36),
WORKSHOPADDRESS_VUCALIBRATIONDATA(36),
WORKSHOPCARDNUMBER_VUCALIBRATIONDATA(18),
WORKSHOPCARDEXPIRYDATE(4),
VEHICLEIDENTIFICATIONNUMBER(17),
VEHICLEREGISTRATIONIDENTIFICATION_TREP5(15),
WVEHICLECHARACTERISTICCONSTANT(2),
KCONSTANTOFRECORDINGEQUIPMENT(2),
LTYRECIRCUMFERENCE(2),
TYRESIZE(15),
AUTHORISEDSPEED(1),
OLDODOMETERVALUE(3),
NEWODOMETERVALUE(3),
OLDTIMEVALUE_VUCALIBRATIONDATA(4),
NEWTIMEVALUE_VUCALIBRATIONDATA(4),
NEXTCALIBRATIONDATE(4)

Adding all the numbers...you will get 167
I used this to develop a tachograph reader section for my app and tested on a lot of files. It is working very well.

**Comment by nyckysoft** [comment]
The specs can be found here
https://dtc.jrc.ec.europa.eu/iot_doc/EU%202016-799-EN.pdf

**Comment by AskmeKolaric** [comment]
Thank you so much.
I have an android app:  https://play.google.com/store/apps/details?id=rs.tahograf 
Friend oy mine maid it. I am now learning all about it. 
If you want to test it out, let me know. 

We have a problem with some cards ( from 30 cards, 4 we cannot read), that is why I am doing all the research not to find the solution. 


By any chance, do you have any .ddd file generation 2?

**Comment by nyckysoft** [comment]
No. don't have generation 2 ddd files... yet. If I get some I will let you know
